quill-core/src/main/scala/io/getquill/norm/RenameProperties.scala

```diff
@@ -80,7 +80,7 @@ object RenameProperties extends StatelessTransformer {
           case (a, schemaA) =>
             val replaceA = replacements(iA, schemaA)
             val onr = BetaReduction(on, replaceA: _*)
-            (FlatJoin(typ, a, iA, onr), Tuple(List(schemaA)))
+            (FlatJoin(typ, a, iA, onr), schemaA)
```

Comment:
`replacements` function later prepends `x._1.` to path hence BetaReduction could not properly replace aliases